code-challenges.js

```diff
@@ -9,67 +9,207 @@
+//console.log(function(string)); to test
+
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
+const vowelRem = (string) => {
+  let stringLetters = string.split("")
+  return stringLetters.filter(letter => {
+    return letter !== "a" && letter !== "A" && letter !== "e" && letter !== "E" && letter !== "i" && letter !== "I" && letter !== "o" && letter !== "O" && letter !== "u" && letter !== "U"
+  }).join("")
+}
+console.log(vowelRem(stringWithVowels1));
+console.log(vowelRem(stringWithVowels2));
 
 
 // --------------------5) Copy/paste your code from #4. Refactor your code to include non-string edge cases. Inform your user that the variable passed into the vowel removal function is not a string.
 // Use the test variables provided below. Expected output: "true is not a string" "42 is not a string" "mCdngMstr"
 
+//Pseudocode
+//create a function
+//argument will be a variable
+//use if else to check if input is string or otherwise
+//if something other than string return string saying input is not a string
+//if string is input then complete following procedure to remove vowels
+//break up letters into an array using .split
+//using newly split array, use .filter to sort out vowels
+//.filter will return an array, so use .join to turn array into a string
+//return new string
+//console.log(function(variable)); to test
+
+
 var refactorTester1 = true
 var refactorTester2 = 42
 var refactorTester3 = "IAmACodingMaster"
 
-
+const vowelRem2 = (input) => {
+  if(typeof input === "string"){
+    let stringLetters = input.split("")
+    return stringLetters.filter(letter => {
+      return letter !== "a" && letter !== "A" && letter !== "e" && letter !== "E" && letter !== "i" && letter !== "I" && letter !== "o" && letter !== "O" && letter !== "u" && letter !== "U"
+    }).join("")}
+  else{
+    return `${input} is not a string`
+  }
+}
+console.log(vowelRem2(refactorTester1));
+console.log(vowelRem2(refactorTester2));
+console.log(vowelRem2(refactorTester3));
 
 
 
 // --------------------6) Create a function that takes in an array of strings and returns the string with the most characters.
 // Expected output: "chopsticks" "stemware"
-
-var utensils = ["fork", "knife", "tongs", "chopsticks", "skewer"]
-var vessels = ["cup", "mug", "stemware", "glass", "tumbler", "tea cup"]
-
-
-
+//
+//Pseudocode
+//create a function
+//argument will be an array
+//use .map and .length to create new array with lengths of strings inside as numbers
+//extract first element from new array
+//compare length to previous record length
+//if larger record index and length of long word
+//if smaller ignore
+//repeat cycle until all elements have been compared
+//return the recorded biggest index
+//console.log(function(array)); to test
+//
+ var utensils = ["fork", "knife", "tongs", "chopsticks", "skewer"]
+ var vessels = ["cup", "mug", "stemware", "glass", "tumbler", "tea cup"]
+
+const bigString = (array) => {
```

Comment:
```suggestion
const getLongestWord = (array) => {
```

function naming will help with communicating intent